maven-plugin/src/main/java/org/openmrs/maven/plugins/CreateProject.java

```diff
@@ -447,6 +450,20 @@ private void createOwaProject() throws MojoExecutionException {
         }
     }
 
+    private void addHelperScripts(String path) {
+        File npmCmd = new File(path, "npm.cmd");
+        URL npmCmdSrc = getClass().getClassLoader().getResource("npm.cmd");
+        File npm = new File(path, "npm");
+        URL npmSrc = getClass().getClassLoader().getResource("npm");
+        try {
+            FileUtils.copyURLToFile(npmSrc, npm);
+            FileUtils.copyURLToFile(npmCmdSrc, npmCmd);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Never do this. You can change your IDE to use new RuntimeExcetion(e); in the try catch template.